ep/build-velox/src/get_velox.sh

```diff
@@ -137,7 +137,7 @@ function process_setup_centos8 {
   sed -i '/^  run_and_time install_fbthrift/a \  run_and_time install_openssl' scripts/setup-centos8.sh
 
   if [ $ENABLE_HDFS == "ON" ]; then
-    sed -i '/^function install_gflags.*/i function install_libhdfs3 {\n cd "\${DEPENDENCY_DIR}"\n github_checkout oap-project/libhdfs3 master\n cmake_install\n}\n' scripts/setup-centos8.sh
+    sed -i '/^function install_gflags.*/i function install_libhdfs3 {\n cd "\${DEPENDENCY_DIR}"\n github_checkout oap-project/libhdfs3 master\n git apply ../../../src/modify_libhdfs3.patch\n cd ..\n cmake_install\n}\n' scripts/setup-centos8.sh
```

Comment:
can we use `CURRENT_DIR` ?

Reply:
Yes, it seems` CURRENT_DIR` is deprecated.
Removed it. Thanks.